### PR TITLE
Added the MIT license file.

### PR DESCRIPTION
**Pull request by slindenberg**
Hello everyone,

I would like to add the MIT license file. We want to use the library in a project for a big company, where it is neccessary that the license is self contained.

Thank you!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2017 Google, Inc.
```
Comment (dbfannin) [inline LICENSE]: Please update Copyright to 
"Copyright (c) 2018 David Fannin"

**Comment by dbfannin** [comment]
@slindenberg, please update the version to 2.0.6. Also, there are 2 package.json files. one at the root, and one under the source directory. Can you please make sure both package.json files have 2.0.6 as the version.  Thanks for contributing!

**Comment by slindenberg** [comment]
I changed it according to your review. Can you check it again, please?

**Comment by dbfannin** [comment]
@slindenberg Thanks for making those changes. it's been merged and published.

**Pull request merged**